Tests/SwiftGRPCTests/ChannelCrash/Generated/channelCrash.grpc.swift

```diff
@@ -0,0 +1,108 @@
+//
```

Comment:
Could you use the existing generated code for `EchoService` instead, and move `ChannelCrashTests.swift` into the `SwiftGRPCTests` folder?